keyboards/jnao/keymaps/bakingpy/keymap.c

```diff
@@ -0,0 +1,204 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define _MAC 0
```

Comment:
Replacing Lines 5-11 with:

```c
enum layer_names {
    _MAC,
    _WINDOWS,
    _TESTMODE,
    _LOWER,
    _RAISE,
    _FKEYS,
    _ADJUST,
};
```

acheives two things:

1. Makes the layers contiguous in the finished firmware, which reduces its flash size.
2. Makes changes to the layers easier to manage. (Add, delete, or edit a line, instead of changing a bunch of hard `#define`s.)